### PR TITLE
Update HDFS Version to 0.1.1

### PR DESCRIPTION
**Pull request by elingg**
In version 0.1.1, we have

-Fixes for task reconciliation/failure handling
-Updated configurations for DCOS and default configurations
-Updated readme

### CODE REVIEW

**Comment by elingg** [comment]
@adam-mesos, please take a look, thanks

**Comment by adam-mesos** [comment]
Not sure why you bumped up the resources in the README, but I don't object.
LGTM. :rocket:

**Comment by elingg** [comment]
Thanks @adam-mesos! can you double check my latest readme changes? Once this is shipped, our builds will be automatically run in teamcity and the S3 buckets will get updated for each released version. :+1:

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Prerequisites
 --------------------------
 1. Install `maven`, `tar`, `unzip`, `wget` in your build host. Set proxy for maven and wget if needed
 2. Install `curl` for all hosts in cluster
+3. `$JAVA_HOME` needs to be set on the host running your HDFS scheduler. This can be set through specifying environment variables in Marathon.
```
Comment (adam-mesos) [inline README.md]: Or, just use `export JAVA_HOME=/path/to/jre` if you're not running the scheduler via Marathon. Let's not force unnecessary dependencies.

File: README.md [hunk 2]
```diff
@@ -25,7 +26,7 @@ Installing HDFS-Mesos on your Cluster
...
 
 ### Mesos-DNS custom configuration
-You can see the example configuration in the `example-conf/mesos-dns` directory. Since Mesos-DNS provides native bindings for master detection, we can simply use those names in our mesos and hdfs configurations. The example configuration assumes your Mesos masters and your zookeeper nodes are colocated. If they aren't you'll need to specify your zookeeper nodes separately.
+You can see the example configuration in the `example-conf/dcos` directory. Since Mesos-DNS provides native bindings for master detection, we can simply use those names in our mesos and hdfs configurations. The example configuration assumes your Mesos masters and your zookeeper nodes are colocated. If they aren't you'll need to specify your zookeeper nodes separately. Also, note that you are using the example in `example-conf/dcos`, the `mesos.hdfs.native-hadoop-binaries` property needs to be set to `false` if your HDFS binaries are not predistributed.
```
Comment (adam-mesos) [inline README.md]: Are we getting rid of the `mesos-dns` example config in favor of a consolidated `dcos` example config? If so, we should remove the mesos-dns config directory.
s/note that you are using/note that if you are using/

**Comment by adam-mesos** [comment]
Just a couple of minor comments. Do we still need two example-conf's?

File: README.md (hunk 1 above)
Comment (elingg) [inline README.md]: Yep, the easiest way is just setting it on the host. Updated the readme.

File: README.md (hunk 2 above)
Comment (elingg) [inline README.md]: True, I don't think we need the other example config anymore.

**Comment by elingg** [comment]
Thanks @adam-mesos, addressed your comments. Can you take a final look?

**Comment by adam-mesos** [comment]
Ship It! :rocket: 
Do we need an "upgrade-guide" that explains that you should tear down your entire 0.1.0 cluster and spin up a new 0.1.1 cluster? Or is there a smoother upgrade path?

**Comment by elingg** [comment]
Hey @adam-mesos, so that depends if you want to upgrade the scheduler or scheduler and executors. If you want to do both, you do have to tear down it all. We definitely need a smoother process. I'm creating  a JIRA for this based on https://github.com/mesosphere/hdfs/issues/23.

**Comment by adam-mesos** [comment]
Did we change anything in the scheduler->executor API that would prevent upgrading only the scheduler?

**Comment by elingg** [comment]
Nothing since release 0.1.0

**Pull request merged**